### PR TITLE
Scream sounds and stuff

### PR DESCRIPTION
**Pull request by tortellinitony**
Disables cat tails and ears

### CODE REVIEW

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -135,6 +135,28 @@
 	can_adjust = 0
 	resistance_flags = 0
 
+/obj/item/clothing/under/cosby
```
Comment (Carbonhell) [inline code/modules/clothing/under/miscellaneous.dm:138]: where is the hitsound for this?and is force = 0.001 still needed?

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -29,23 +29,24 @@
...
 /datum/emote/living/carbon/human/fart/run_emote(mob/user, params)
+	message = null
 	if(user.stat != CONSCIOUS)
 		return
 	var/obj/item/organ/internal/butt/B = user.getorgan(/obj/item/organ/internal/butt)
 	if(!B)
 		user << "\red You don't have a butt!"
 		return
-	var/lose_butt = prob(6)
+	var/lose_butt = prob(22)
```
Comment (Carbonhell) [inline code/modules/mob/living/carbon/human/emote.dm:39]: why so high?

Comment (tortellinitony) [reply code/modules/mob/living/carbon/human/emote.dm:39]: it was taking way too many farts before

File: code/modules/clothing/under/miscellaneous.dm (hunk 1 above)
Comment (tortellinitony) [reply code/modules/clothing/under/miscellaneous.dm:138]: I don't know about the force but I'll see if I can add the hitsound

File: code/game/objects/items.dm [hunk 3]
```diff
@@ -32,6 +32,7 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 	pass_flags = PASSTABLE
 	pressure_resistance = 4
 	var/obj/item/master = null
+	var/alternate_screams  = list()
```
Comment (Carbonhell) [inline code/game/objects/items.dm:35]: var/list/alternate_screams

**Review by Carbonhell — CHANGES_REQUESTED** [review]

File: code/game/objects/items.dm (hunk 3 above)
Comment (tortellinitony) [reply code/game/objects/items.dm:35]: this is different

**Pull request merged**